[12.0] [MIG] maintenance_equipment _status

NuriaXifre opened the pull request:
MIgration of the module maintenance_equipment_status to v12
cc ~ @ForgeFlow

JordiBForgeFlow committed:
[11.0][ADD] maintenance_equipment_status

oca-travis committed:
[UPD] Update maintenance_equipment_status.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

oca-travis committed:
[UPD] Update maintenance_equipment_status.pot

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (22 of 22 strings)

Translation: maintenance-11.0/maintenance-11.0-maintenance_equipment_status
Translate-URL: https://translation.odoo-community.org/projects/maintenance-11-0/maintenance-11-0-maintenance_equipment_status/es/

oca-travis committed:
[UPD] Update maintenance_equipment_status.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-11.0/maintenance-11.0-maintenance_equipment_status
Translate-URL: https://translation.odoo-community.org/projects/maintenance-11-0/maintenance-11-0-maintenance_equipment_status/

etobella reviewed (approved):
No major changes. LGTM :+1:

etobella commented:
Well, it is crazy to work with scrap, because two headers will be generated then..., maybe this header needs to be added on base_maintenance?
@lreficent

LoisRForgeFlow commented on maintenance_equipment_status/views/maintenance_equipment_views.xml:
```diff
@@ -0,0 +1,48 @@
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form"/>
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <header>
```
@etobella You mean this header?

etobella replied:
Yes

LoisRForgeFlow replied:
Ok, I agree.

@NuriaMartinXifre can you migrate base_maintenance and include the empty header for this view in that module? then, we'll need to include it here as a dependency and extend the view accordingly.

etobella commented:
@NuriaMForgeFlow Fix the commit history please, as you have some merges on it and a `syntax fix`

etobella committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (22 of 22 strings)

Translation: maintenance-11.0/maintenance-11.0-maintenance_equipment_status
Translate-URL: https://translation.odoo-community.org/projects/maintenance-11-0/maintenance-11-0-maintenance_equipment_status/es/

etobella commented:
@NuriaMForgeFlow Something weird happened with the commit history. Can you review it please?

etobella commented:
Something weird happened with the commit 288676d7ca6bca1ffb431e81c1a207008b230d4d

oca-travis committed:
[UPD] Update maintenance_equipment_status.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-11.0/maintenance-11.0-maintenance_equipment_status
Translate-URL: https://translation.odoo-community.org/projects/maintenance-11-0/maintenance-11-0-maintenance_equipment_status/

NuriaXifre committed:
[12.0][MIG] maintenance_equipment_status

etobella commented:
Just a question, should we need to merge all the translation commits?
@LoisRForgeFlow

LoisRForgeFlow commented:
@etobella we can squash them together if you want

etobella commented:
Let's keep it this way
/ocabot merge
